lib/i18n/tokenizer/base.ts

```diff
@@ -524,7 +530,7 @@ export default class BaseTokenizer {
         });
     }
 
-    tokenize(text : string) : TokenizerResult {
+    tokenize(text : string, preserved ?: RegExp) : TokenizerResult {
```

Comment:
What's this about?

Reply:
If we want to make sure a token doesn't get tokenized - similar to special_tokens in huggingface tokenizers.
(I found it useful to keep entities such as "PHONE_NUMBER" intact )